ruby/README.md

```diff
@@ -233,3 +280,5 @@ set shiftwidth=2
 ```
 "rulers": [ 80 ]
 ```
+
+[external configuration]: https://codeclimate.com/changelog/582495c32c33066f1b00191d
```

Comment:
wdyt about linking to the [doc](https://docs.codeclimate.com/v1.0/docs/configuring-the-prepare-step) instead of the changelog?

I think that makes sense for internal policy but nbd either way - easy to get from changelog to doc in any case, so up to you!